lib/datadog/appsec/contrib/sinatra/integration.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require_relative '../../../tracing/contrib'
```

Comment:
I'm not sure about this `require` here: this integration file doesn't itself seem to need it.